spec/rack/profiler_spec.rb

```diff
@@ -193,6 +194,27 @@ def trigger_dummy_event(name, payload = {})
           parsed_body['events'].map { |e| e['name'] }
         ).to include('rack-profiler.total_time', 'rack-profiler.step')
       end
+
+      context "when authorization is configured" do
+        before do
+          profiler.authorize { |env| env['rack-profiler-allowed'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.